rpc/grpc/client/errors.go

```diff
@@ -0,0 +1,54 @@
+package client
+
+import "fmt"
+
+type ErrBlockResults struct {
+	Height int64
+	Err    error
+	latest bool
+}
+
+func (e ErrBlockResults) Error() string {
+	if e.latest {
+		return fmt.Sprintf("error fetching BlockResults for latest height :: %s", e.Err.Error())
```

Comment:
What's with the fancy punctuation here?

```suggestion
		return fmt.Sprintf("error fetching BlockResults for latest height: %s", e.Err.Error())
```

Reply:
Reverted.